features/steps/static_primary.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+from behave import step
+from time import sleep
+
+
+@step('I start {name:w} as static primary')
+def start_patroni_with_static_primary(context, name):
+    return context.pctl.start(name, custom_config={'bootstrap': {'dcs': {'static_primary': name}}})
+
+
+@step('I start {name:w} with a configured static primary will not boot after {time_limit:d} seconds')
+def start_patroni_as_replica_with_static_primary(context, name, time_limit):
+    return context.pctl.start_with_expected_failure(name, max_wait_limit=time_limit)
+
+
+@step('"{name}" key not in DCS after waiting {time_limit:d} seconds')
+def check_member_not_present(context, name, time_limit):
+    sleep(time_limit)
+    found_value = False
+    try:
+        res = json.loads(context.dcs_ctl.query(name))
+        if res is not None:
+            found_value = True
+    except Exception:
+        pass
+
+    if found_value:
+        assert False, "found value under DCS key {} after {} seconds".format(name, time_limit)
```

Comment:
1. Member keys will not expire because they are updated by running Patroni
2. In case of K8s it is even worse, because the info is stored in Pod annotations.